tools/autograd/load_derivatives.py

```diff
@@ -315,6 +316,12 @@ def saved_variables(formula, args):
             'suffix': '_dim',
             'type': 'int64_t',
         }),
+        # replace self.strides() with self_strides
+        (r'{}.strides\(\)', {
+            'suffix': f'_{"_".join(var_names)}_strides',
```

Comment:
Note that I decided to include the variable names in the replacement suffix, just in case the derivative formulas for two different inputs call `.strides()` on the same tensor. For instance:

```
- name: example_func ...
  input1: ... self.strides() ...
  input2, input3: ... self.strides() ...
```

The replacement for this case would be:

```
grad_fn->self_input1_strides = backward_strides_or_error(self, "self", TensorList(input1), "input1")
grad_fn->self_input2_input3_strides = backward_strides_or_error(self, "self", TensorList(input2, input3), "input2, input3")
```

We don't currently have any cases like this. But if we decided not to have two replacements like this, and someone writes a derivative formula similar to the above, they would have a difficult time figuring out why it fails and how to fix it.

Reply:
Should that be added to all the replacements done in this list then?

Reply:
I wouldn't say so. The only reason I was including the variable names in the suffix was because the `expr` is different for different sets of variable names (specifically, the `backward_strides_or_error` would be called with different arguments). None of the other replacements needs the variable names in the suffix because their `expr` does not depend on the variable names.

Now that I'm changing the strides replacements to only support single derivatives of the same tensor that `strides()` is being called on, we will no longer need the variable names in the strides suffix either.